src/frontend/src/components/common/Pagination/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { faAngleLeft } from '@fortawesome/free-solid-svg-icons';
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons';
+
+type PaginationProps = {
+  currentPage: number;
+  totalEntries: number;
+  pageSize: number;
+  changePage: (page: number) => void;
+  changePageSize: (pageSize: number) => void;
+};
+
+const Pagination = ({
+  currentPage,
+  totalEntries,
+  pageSize,
+  changePage,
+  changePageSize,
+}: PaginationProps) => {
+  const PageSizeOptions = [10, 20, 30, 50, 100];
+  return (
+    <div className="pagination">
+      <div className="pagination__container">
+        <div className="pagination__container__left">
+          <div className="pagination__container__left__text">
+            <span>
+              Showing {(currentPage - 1) * pageSize + 1} to{' '}
+              {totalEntries < pageSize * currentPage
+                ? totalEntries
+                : currentPage * pageSize}{' '}
+              of {totalEntries} entries
+            </span>
+          </div>
+        </div>
+        <div className="pagination__container__right">
+          <div className="pagination__container__right__text">
+            <span className="rows-per-page">
+              Rows per page
+              <div className="dropdown">
```

Comment:
why are we not using our common component for Dropdown?

Reply:
I think it is not bootstrap based and there are a few customization that need to be resolved. Afraid to modify as it might break existing functionality.  TODO to make the dropdown more of a common component.